x-pack/test/cases_api_integration/common/fixtures/plugins/security_solution/server/plugin.ts

```diff
@@ -33,7 +33,10 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
         all: {
           app: ['kibana'],
           cases: {
-            all: ['securitySolutionFixture'],
```

Comment:
To make this minimal, we need to remove the `all` since that'd override everything and give all. This should only include the things that aren't placed in the sub features section (everything except `delete`).